docs/versionIndex.md

```diff
@@ -12,6 +12,7 @@ backend services have been tested with each released version of ADF.
 
 ## Versions
 
+-   [v6.4.0](#v640)
```

Comment:
Is that version proper? In package.json there is actually 6.3.0 
https://github.com/Alfresco/alfresco-ng2-components/blob/develop/package.json

Reply:
Yes but this is the version we already released and now we're producing alphas for `6.4.0`. So this interface will be available there